### ISSUE TITLE
Image2StyleGAN: How to Embed Images Into the StyleGAN Latent Space?

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#105]
# TL;DR
A paper that does with various experiments in the latent space of StyleGAN. They found that there are characteristics such as, It is possible to reproduce images other than faces after they are put into latent space, and it is weak at data augmentation such as rotation,  you try to get an intermediate representation between image and image by mixing latent space, you will get a face image in the middle. 
![image](https://user-images.githubusercontent.com/24524018/92562513-0ed20700-f2b1-11ea-8d26-3fae69965e99.png)

# Why it matters:

# Paper URL
http://openaccess.thecvf.com/content_ICCV_2019/papers/Abdal_Image2StyleGAN_How_to_Embed_Images_Into_the_StyleGAN_Latent_Space_ICCV_2019_paper.pdf 

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
